components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -240,6 +240,17 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         }
     }
 
+    /**
+     * Tries to return a session token
+     *
+     * @throws FxaException Will send you an exception if there is no session token set
+     */
+    fun getSessionToken(): String {
```

Comment:
it works! 
![image](https://user-images.githubusercontent.com/128755/64987479-432a0780-d897-11e9-8b50-468b5e8d3d57.png)